src/main/java/core/basesyntax/dao/UserDaoImpl.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.lib.Dao;
+import core.basesyntax.model.User;
+import java.util.List;
+
+@Dao
```

Comment:
I'm not sure, but maybe we should put @Dao annotation on UserDao interface, not on UserDaoImpl?

Reply:
You can have a lots of implementations of your interface and @Dao should point to the specific implementation that you want to be used. So we should put it to implementation  class.